Logger/FileLogger.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.IO;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        private string outPath;
```

Comment:
Fields should be underscore PacalCase
```suggestion
        private string _OutPath;
```